src/benchmarks/micro/common.props

```diff
@@ -4,8 +4,8 @@
     <UseSharedCompilation>false</UseSharedCompilation>
     <!-- The Python script can narrow down the TFMs to what the user has asked for -->
     <TargetFrameworks>$(PYTHON_SCRIPT_TARGET_FRAMEWORKS)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetFrameworks)' == '' AND '$(OS)' == 'Windows_NT'">net461;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
```

Comment:
Because we had a "bug" (which BTW I introduced some time ago ;p )

```xml
<TargetFrameworks>$(PYTHON_SCRIPT_TARGET_FRAMEWORKS)</TargetFrameworks>
<TargetFrameworks Condition="'$(TargetFrameworks)' == ''">netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
<TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks> <!-- not taking PYTHON_SCRIPT_TARGET_FRAMEWORKS into account -->
```

when we were running from the Python script on Windows, we were always restoring everything for what the user has asked **AND** net461. Example: `--framework netcoreapp3.0" was 

```xml
<TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
```

with this change we restore only what we have asked for and save some time on Windows when using the scripts

Reply:
Good call, thanks for the explanation.